Migrations/Schema/v1_1/WebhookBundleMigration.php

```diff
@@ -1,30 +1,25 @@
 <?php
 
-namespace Aligent\AsyncBundle\Migrations\Schema\v1_1;
+namespace Aligent\AsyncEventsBundle\Migrations\Schema\v1_1;
 
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\DBAL\Types\TextType;
-use Oro\Bundle\MigrationBundle\Migration\QueryBag;
 use Oro\Bundle\MigrationBundle\Migration\Migration;
+use Oro\Bundle\MigrationBundle\Migration\QueryBag;
 
 /**
- * Class UpdateToTextField
+ * Class WebhookBundleMigration
  *
  * @category  Aligent
- * @package   Aligent\AsyncBundle\Migrations\Schema\v1_1
+ * @package   Aligent\WebhookBundle\Migrations\Schema\v1_0
  * @author    Adam Hall <adam.hall@aligent.com.au>
  * @copyright 2020 Aligent Consulting.
  * @link      http://www.aligent.com.au/
  */
-class UpdateToTextField implements Migration
+class WebhookBundleMigration implements Migration
 {
     /**
-     * Modifies the given schema to apply necessary changes of a database
-     * The given query bag can be used to apply additional SQL queries before and after schema changes
-     *
-     * @param Schema $schema
-     * @param QueryBag $queries
-     * @return void
+     * @inheritDoc
      */
     public function up(Schema $schema, QueryBag $queries)
```

Comment:
@AdamJHall Do you think I can remove this migration at all and just change the column type in the initial create table migration?

Reply:
Yes that should be fine :)